From 49199a78bfbcc64aa0cdb5b4cb3b5504242862dd Mon Sep 17 00:00:00 2001
From: Ismael Luceno <ismael@iodev.co.uk>
Date: Fri, 20 Jun 2025 20:02:10 +0200
Subject: [PATCH] binutils: Disable gprofng

It contains unportable code that fails to build against implementations
other than glibc.

Status: Pending
Origin: Source Mage
Signed-off-by: Ismael Luceno <ismael@sourcemage.org>
---
 scripts/build/binutils/binutils.sh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/build/binutils/binutils.sh b/scripts/build/binutils/binutils.sh
index b16be9041662..1fa8538c87c8 100644
--- a/scripts/build/binutils/binutils.sh
+++ b/scripts/build/binutils/binutils.sh
@@ -167,6 +167,9 @@ do_binutils_backend() {
         extra_config+=("--disable-multilib")
     fi
 
+    # Disable gprofng, it contains unportable code
+    extra_config+=("--disable-gprofng")
+
     # Disable gdb when building from the binutils-gdb repository.
     extra_config+=("--disable-sim")
     extra_config+=("--disable-gdb")
-- 
2.49.0

